modules/components/pages/processors/gcp_vertex_ai_chat.adoc

```diff
@@ -0,0 +1,207 @@
+= gcp_vertex_ai_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+Generates responses to messages in a chat conversation, using the Vertex AI API.
```

Comment:
consider linking to https://cloud.google.com/vertex-ai/docs/start/introduction-unified-platform